js/count-chars.js

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
I'm still concerned about adding this, and would much rather create a different Issue/PR since it essentially affects the end result. Plus, tried it before and had to remove it. https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/1767/files

Reply:
...but that did make me notice I forgot to rename aioseop_post_edit. Since it's not just for post edit now.

Reply:
@EkoJR not sure how it affects the end result. Am I missing something here?

Reply:
Yes, a different Issue/PR after this one gets merged. We've already been through this before where we had to go back and strip out code that was added into a PR in the same manner (link above was when it was removed). I would much rather have a seperate PR merge/squash into the main branches. Makes repo more organized and easier to handle if it does go wrong again.

Reply:
@michaeltorbert do you want to create a separate issue for this? Maybe something that is relevant to all JS files

Reply:
@contactashish13 @EkoJR Sure a separate issue is fine. You can go ahead and do that.